### PR TITLE
Add some frontend packages to CircleCI container and allow editing of php-fpm/nginx conf

### PR DESCRIPTION
**Pull request by larowlan**

### CODE REVIEW

File: php/circleci-decoupled/fpm/php-fpm.conf [hunk 1]
```diff
@@ -0,0 +1,27 @@
+[global]
+error_log = /proc/self/fd/2
+
+[www]
+listen = [::]:${PHP_FPM_PORT}
+
+chdir = /data/api/app
```
Comment (kimpepper) [inline php/circleci-decoupled/fpm/php-fpm.conf:7]: A lot of duplication here. Is this the only difference? Could this just be added as an included file?

File: php/circleci-decoupled/nginx/default.conf [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+}
+
+map $host$uri $redirectdomain {
+  include /etc/nginx/redirects.conf;
+}
+
+server {
+    listen 8080;
+
+    root /data/api/app;
```
Comment (kimpepper) [inline php/circleci-decoupled/nginx/default.conf:23]: Ditto

**Comment by nickschuch** [comment]
I'm happy with the size of the commit, a little wary of adding more images. But.... lets revisit once we have things running.

File: php/circleci-decoupled/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+RUN apk --update --no-cache add ca-certificates \
+  g++ \
+  git \
+  openssh-client \
+  python2 \
+  autoconf \
+  automake \
+  libpng-dev \
+  libtool \
+  nasm
```
Comment (kimpepper) [inline php/circleci-decoupled/Dockerfile:20]: Some of these packages could be added to php-circleci?

Comment (larowlan) [reply php/circleci-decoupled/Dockerfile:20]: @kimpepper Happy for me to add all of them?

Comment (kimpepper) [reply php/circleci-decoupled/Dockerfile:20]: Second thoughts, just leave it for now. We'll circle :wink:  back later.

**Review by kimpepper — APPROVED** [review]
:+1:

**Pull request merged**